src/impl/select.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import has from 'lodash/get';
 import range from 'lodash/range';
```

Comment:
As I investigate, lodash/range also pull in too many dependencies, can we try to simplify it as well like arrayShuffle?